### ISSUE TITLE
f mod error

### ISSUE THREAD

**Issue by gwelw** [redstone/LegacyFactions#16]
Error on run /f mod without arguments
```
[16:55:53] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'f' in plugin LegacyFactions v1.3.0
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at org.bukkit.craftbukkit.v1_11_R1.CraftServer.dispatchCommand(CraftServer.java:650) ~[spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.PlayerConnection.handleCommand(PlayerConnection.java:1354) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:1189) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_131]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_131]
	at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:747) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:399) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:678) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:576) [spigot.jar:git-Spigot-3fb9445-6e3cec8]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_131]
Caused by: java.lang.NullPointerException
	at java.lang.String.replace(Unknown Source) ~[?:1.8.0_131]
	at net.redstoneore.legacyfactions.cmd.FCommand.strAsFPlayer(FCommand.java:149) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.FCommand.strAsBestFPlayerMatch(FCommand.java:169) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.FCommand.argAsBestFPlayerMatch(FCommand.java:173) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.FCommand.argAsBestFPlayerMatch(FCommand.java:177) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.FCommand.argAsBestFPlayerMatch(FCommand.java:181) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.CmdFactionsMod.perform(CmdFactionsMod.java:31) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.MCommand.execute(MCommand.java:119) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.FCommand.execute(FCommand.java:45) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.MCommand.execute(MCommand.java:105) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.FCommand.execute(FCommand.java:45) ~[?:?]
	at net.redstoneore.legacyfactions.cmd.MCommand.execute(MCommand.java:123) ~[?:?]
	at net.redstoneore.legacyfactions.FactionsPluginBase.handleCommand(FactionsPluginBase.java:184) ~[?:?]
	at net.redstoneore.legacyfactions.FactionsPluginBase.handleCommand(FactionsPluginBase.java:147) ~[?:?]
	at net.redstoneore.legacyfactions.Factions.handleCommand(Factions.java:227) ~[?:?]
	at net.redstoneore.legacyfactions.Factions.onCommand(Factions.java:236) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-3fb9445-6e3cec8]
	... 15 more
```

**Comment by MarkehMe** [comment]
Thanks for reporting!

**Comment by ctooley17** [comment]
Fixed in #17

**Comment by MarkehMe** [comment]
Fixed by @ctooley17  🎉  

This will be available in the next update.
